draft-ietf-quic-multipath.md

```diff
@@ -1487,68 +1487,65 @@ TBD-13 (experiments use 0x004e4f5f4349445f) | NO_CID_AVAILABLE | Path abandoned
 
 # Security Considerations
 
-The multipath extension retains all the security features of {{QUIC-TRANSPORT}} and {{QUIC-TLS}}
-but requires some additional consideration regarding the following amendments:
+The multipath extension retains all security properties of {{QUIC-TRANSPORT}} and {{QUIC-TLS}}
+but requires some additional consideration regarding:
 
-- the need of potential additional resources as connection IDs are now maintained per-path;
-- the provisioning of multiple concurrent path contexts and the associated resources;
-- the possibility to create and use multiple simultaneous paths and the corresponding increased amplification risk for request forgery attacks;
-- the changes on encryption requirements due to the use of multiple packet number spaces.
+- potential additional resource usage for per-path connection IDs and multiple concurrent path contexts;
+- a potentially increased amplification risk for request forgery attacks if multiple paths are used simultaneously;
+- changes to the nonce calculation due to the use of multiple packet number spaces.
 
 
 ## Memory Allocation for Per-Path Resources
 
-The initial_max_path_id transport parameter and the Max Path ID field
-in the MAX_PATH_ID frame limit the number of paths an endpoint is willing
-to maintain and accordingly limit the associated path resources.
-
+The maximum Path ID limit in initial_max_path_id or MAX_PATH_ID frame
+limit the number of paths an endpoint is willing
+to maintain and accordingly thereby also limit the associated path resources.
 Furthermore, as connection IDs have to be issued by both endpoints for the
-same path ID before an endpoint can open a path, each endpoint can further
-control the per-path resource usage (beyond the connection IDs) by limiting
-the number of Path ID that it issues connection IDs for.
+same Path ID before an endpoint can open a path, each endpoint could also
+control the per-path resource usage by only
+issuing connection IDs for a limited number of paths. However, using
+the maximum Path ID limit in initial_max_path_id or the MAX_PATH_ID frame is preferred.
 
-Therefore, to avoid unnecessarily resource usage, that potentially could be exploited
-in a resource exhaustion attack, endpoints should allocate those additional path resource,
+To avoid unnecessarily resource usage, that potentially could be exploited
+in a resource exhaustion attack, endpoints SHOULD allocate additional path resources,
```

Comment:
Suggestion:

To avoid unnecessarily resource usage that could be exploited

Fewer words, and also no comma before "that"